instrumentation/http-url-connection/javaagent/src/test/groovy/SpringRestTemplateTest.groovy

```diff
@@ -28,22 +27,28 @@ class SpringRestTemplateTest extends HttpClientTest implements AgentTestTrait {
   }
 
   @Override
-  int doRequest(String method, URI uri, Map<String, String> headers = [:]) {
+  Void buildRequest(String method, URI uri, Map<String, String> headers) {
+    return null
```

Comment:
Should we set `testReusedRequest` to false to be explicit about it not being tested (I guess it's not being tested)?

Reply:
pushed a new commit that seems a bit better, lmk